spec/requests/v0/health_care_applications_spec.rb

```diff
@@ -28,6 +29,22 @@
       )
     end
 
+    context 'hca_disable_bgs_service enabled' do
+      before do
+        Flipper.enable(:hca_disable_bgs_service)
```

Comment:
To ensure other tests do not mess up, you can allow the flipper to be enabled for certain tests. 
` allow(Flipper).to receive(:enabled?).with(: hca_disable_bgs_service).and_return(true)` or 
` allow(Flipper).to receive(:enabled?).with(: hca_disable_bgs_service, instance_of(User)).and_return(true)`